flaky-tests/ci-providers/buildkite-quickstart.md

```diff
@@ -44,16 +47,16 @@ steps:
     key: tests
   - label: "Upload test results"
     commands:
-      - curl -fsSL --retry 3 "UPLOADER_LINK" -o ./trunk-analytics-uploader
-      - chmod +x ./trunk-analytics-uploader
-      - ./trunk-analytics-uploader upload --junit-paths *.xml --org-url-slug trunk --token $$TRUNK_TOKEN
+    - curl -fsSL --retry 3 "CLI_LINK" | tar -xvz > ./trunk-analytics-cli
+      - chmod +x ./trunk-analytics-cli
```

Comment:
Did you test this? The resulting binary should already be executable

Reply:
I did test that the download worked. I didn't test if it was already executable. Does tar.gz preserve the executable bit on mac and linux?

Reply:
I removed the chmod lines.